packages/sentinel_one_cloud_funnel/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.14.1"
+  changes:
+    - description: Add missing event preservation template expansions for GCS input.
+      type: enhancement
```

```suggestion
      type: bugfix
```
I see based on patch version increment and also based on labels that its supposed to be bugfix?